bio/mofaflex/plotting/wrapper.py

```
@@ -0,0 +1,107 @@
+    model_path: Path = Path(snakemake.input[0])
+
+    model: mfl.MOFAFLEX = mfl.MOFAFLEX.load(model_path)
+
+    n_factors: int = snakemake.params.get("n_factors", 15)
+    likelihoods: Literal["Normal", "NegativeBinomial", "Binomial"] = (
+        snakemake.params.get("likelihoods", "Normal")
+    )
+    batch_size: int = snakemake.params.get("batch_size", 1000)
+    seed: int = snakemake.params.get("seed", 42)
+
+    training_curve: Path = Path(snakemake.output.get("training_curve"))
```

Ruff (0.15.17)

[error] Undefined name `snakemake`

(F821)